src/options.cpp

```diff
@@ -2065,79 +2120,89 @@ void options_manager::add_options_graphics()
 
-    add( "ANIMATION_SCT_USE_FONT", "graphics", to_translation( "SCT with Unicode font" ),
+    add( "ANIMATION_SCT_USE_FONT", page_id, to_translation( "SCT with Unicode font" ),
          to_translation( "If true, will display scrolling combat text with Unicode font." ),
          true
        );
 
     get_option( "ANIMATION_SCT_USE_FONT" ).setPrerequisite( "ANIMATION_SCT" );
 
-    add( "ANIMATION_DELAY", "graphics", to_translation( "Animation delay" ),
+    add( "ANIMATION_DELAY", page_id, to_translation( "Animation delay" ),
          to_translation( "The amount of time to pause between animation frames in ms." ),
          0, 100, 10
        );
 
     get_option( "ANIMATION_DELAY" ).setPrerequisite( "ANIMATIONS" );
 
-    add( "BLINK_SPEED", "graphics", to_translation( "Blinking effects speed" ),
+    add( "BLINK_SPEED", page_id, to_translation( "Blinking effects speed" ),
          to_translation( "The speed of every blinking effects in ms." ),
          100, 5000, 300
        );
 
-    add( "FORCE_REDRAW", "graphics", to_translation( "Force redraw" ),
+    add( "FORCE_REDRAW", page_id, to_translation( "Force redraw" ),
          to_translation( "If true, forces the game to redraw at least once per turn." ),
          true
        );
+    });
 
     add_empty_line();
 
-    add( "ENABLE_ASCII_TITLE", "graphics",
+    add_option_group( "graphics", Group( "ascii_opts", to_translation( "ASCII Graphic Options" ),
+                                          to_translation( "Options regarding ASCII graphic." ) ),
+    [&]( const std::string & page_id ) {
+    add( "ENABLE_ASCII_TITLE", page_id,
          to_translation( "Enable ASCII art on the title screen" ),
          to_translation( "If true, shows an ASCII graphic on the title screen.  If false, shows a text-only title screen." ),
          true
        );
 
-    add( "SEASONAL_TITLE", "graphics", to_translation( "Use seasonal title screen" ),
+    add( "SEASONAL_TITLE", page_id, to_translation( "Use seasonal title screen" ),
          to_translation( "If true, the title screen will use the art appropriate for the season." ),
          true
        );
 
     get_option( "SEASONAL_TITLE" ).setPrerequisite( "ENABLE_ASCII_TITLE" );
 
-    add( "ALT_TITLE", "graphics", to_translation( "Alternative title screen frequency" ),
+    add( "ALT_TITLE", page_id, to_translation( "Alternative title screen frequency" ),
          to_translation( "Set the probability of the alternate title screen appearing." ), 0, 100, 10
        );
 
     get_option( "ALT_TITLE" ).setPrerequisite( "ENABLE_ASCII_TITLE" );
 
     add_empty_line();
 
+    add_option_group( "graphics", Group( "term_opts", to_translation( "Terminal Display Options" ),
+                                          to_translation( "Options regarding terminal display." ) ),
+    [&]( const std::string & page_id ) {
     add( "TERMINAL_X", "graphics", to_translation( "Terminal width" ),
```

Comment:
Should replace "graphics" in this group with `page_id`. Also should add closing brackets to below line 2214.